edkrepo_manifest_parser/edk_manifest.py

```diff
@@ -378,7 +394,34 @@ def get_combo_element(self, name):
         for combo in combinations.iter(tag='Combination'):
             if combo.attrib['name'] == name:
                 return copy.deepcopy(combo)
-        raise ValueError(COMB_UNKOWN_ERROR.format(name, self._fileref))
+        raise ValueError(COMB_UNKNOWN_ERROR.format(name, self._fileref))
+
+    def get_all_patchsets(self):

```

Comment:
Please add a doc string to this detailing the format and content of the return value